lib/srv/reexec.go

```diff
@@ -164,12 +164,8 @@ func RunCommand() (io.Writer, int, error) {
 		}
 		errorWriter = tty
 		err = uacc.Open(c.UaccMetadata.UtmpPath, c.UaccMetadata.WtmpPath, c.Login, c.UaccMetadata.Hostname, c.UaccMetadata.RemoteAddr, tty)
-		if err != nil {
-			// the error is critical and we should fail command execution
-			if !trace.IsAccessDenied(err) && !trace.IsNotFound(err) {
-				return errorWriter, teleport.RemoteCommandFailure, trace.Wrap(err)
-			}
-		} else {
+		// uacc support is best-effort, only enable it if Open is successful
+		if err == nil {
```

Comment:
So there's no good way to log this error?

Reply:
Currently the only way is to write it to stdout or stderr, both of which will also contain output of the actual command the user is trying to run, and they will be sent back to the user rather than written to the teleport logs. This doesn't seem like the right thing to do when the command is successful but utmp is unsupported.

With some more effort it would be possible to set up a pipe or something to communicate warning logs to the parent process out-of-band with the command output, let me know if you think this would be worthwhile

Reply:
@nklaassen It's ok, I would just leave a todo item or explanation as to why we're basically ignoring the error here.

Reply:
thanks, I left a comment to that effect